linera-views/src/backends/scylla_db.rs

```diff
@@ -887,6 +890,7 @@ impl TestKeyValueStore for JournalingKeyValueStore<ScyllaDbStoreInternal> {
         let common_config = CommonStoreInternalConfig {
```

Comment:
I'm trying to remove this common config because most options there only work for a subset of the databases